tests/runtime/TestIndirectCall.hip

```diff
@@ -0,0 +1,35 @@
+// Test kernel with an indirect call.
+//
+// NOTES:
+//
+// * Core SPIR-V does not support indirect calls without an extension. In this
+//   case the llvm-spirv inserts SPV_INTEL_function_pointers extension so this
+//   program is only runnable in environments that supports it.
+//
+// * Invalid SPIR-V binary is known to be generated.
```

Comment:
Perhaps link to the SPIRV-translator issue?

Reply:
To be done.